docs/resources/guides/deployment/bare_metal.rst

```diff
@@ -61,6 +61,13 @@ Install the Gel package.
 
    $ sudo yum install gel-6
 
+Disable SELinux.
+
+.. code-block:: bash
+
+   $ sed -i 's/SELINUX=enforcing/SELINUX=disabled/' /etc/selinux/config
+   $ reboot
+
```

Comment:
This will no longer be necessary when https://github.com/geldata/gel-pkg/issues/145 is fixed. If this change is accepted I'll open an issue reminding us to remove this blurb once the gel-pkg issue is fixed.

Reply:
follow up issue https://github.com/geldata/gel/issues/8440